core/src/Cuda/Kokkos_Cuda_Team.hpp

```diff
@@ -344,9 +344,10 @@ class CudaTeamMember {
   // Private for the driver
 
   KOKKOS_INLINE_FUNCTION
-  CudaTeamMember(void* shared, const int shared_begin, const int shared_size,
-                 void* scratch_level_1_ptr, const int scratch_level_1_size,
-                 const int arg_league_rank, const int arg_league_size)
+  CudaTeamMember(void* shared, const size_t shared_begin,
+                 const size_t shared_size, void* scratch_level_1_ptr,
+                 const size_t scratch_level_1_size, const int arg_league_rank,
+                 const size_t arg_league_size)
```

Comment:
why did you change the type of `arg_league_size`?

Reply:
I'll revert that.